Quake/gl_model.c

```diff
@@ -2242,6 +2309,11 @@ void Mod_LoadBrushModel (qmodel_t *mod, void *buffer)
 		break;
 	case BSP2VERSION_BSP2:
 		bsp2 = 2;	//sanitised revision
+		break;
+	case BSPVERSION_QUAKE64:
+		bsp2 = false;
+		mod->bspversion = BSPVERSION_QUAKE64;
+
 		break;
```

Comment:
Down below around line 2341, you should change
`if (!bsp2 && external_vis.value && sv.modelname[0] && !q_strcasecmp(loadname, sv.name))`
to something like:
`if (mod->bspversion == BSPVERSION && external_vis.value && sv.modelname[0] && !q_strcasecmp(loadname, sv.name))`

.. so that external vis file loading is always restricted to traditional maps.